company-clang.el

```diff
@@ -80,6 +80,76 @@ or automatically through a custom `company-clang-prefix-guesser'."
     (insert-file-contents-literally file nil beg end)
     (buffer-string)))
 
+(defcustom company-clang-parse-documentation nil
+  "When non-nil, clang will parse completion's comments/documentation."
+  :type 'boolean)
+
+(defcustom company-clang-documentation-fill-column 70
+  "Column beyond which automatic line-wrapping should happen."
+  :type 'integer)
+
+(defcustom company-clang-documentation-justify 'full
+  "Specifies which  kind of justification to do."
+  :type '(choice (const :tag "Full" full)
+		 (const :tag "Left" left)
+		 (const :tag "Right" right)
+		 (const :tag "Center" center)
+                 (const :tag "None" none)))
+
+(defvar company-clang-summary-list
+  "Association list of tags and them related documentation.")
+
+(defun company-clang-meta-for-tag (tag)
+  "Extract the meta data of a TAG from `company-clang-summary-list'."
+  (let ((element (assoc tag company-clang-summary-list))
+	(match)
+	(meta))
+    (when element
+      (setq match (car element))
+      (setq meta (get-text-property 0 'meta match))
+      meta)))
+
+(defun company-clang-documentation-for-tag (tag)
+  "Extract the documentation of a TAG from `company-clang-summary-list'."
+  (let ((element (assoc tag company-clang-summary-list))
+	(doc))
+    (when element
+      (setq doc (car (cdr element)))
+      doc)))
+
+(defun company-clang-doc-buffer (tag)
+  "Create the documentation buffer for a TAG."
+  (let ((meta (company-clang-meta-for-tag tag))
+	(doc (company-clang-documentation-for-tag tag))
+	(emptylines "\n\n"))
+    (unless (and doc meta)
```

Comment:
When filtering by prefix gives a candidate like `extern` (possible prefix: *ext*), `float` (possible prefix: *flo*), etc. there is no `meta` (no arguments) and also no comments:

**"COMPLETION: extern"**
**"COMPLETION: float"**
etc.

In these cases `meta` is empty.